### DISCUSSION TITLE
Can page designer support SEO friendly URLs?

### DISCUSSION THREAD

**Discussion by nicekiwi** [SalesforceCommerceCloud/sfcc-ci#435] [Team Posts]
There is a field in the page settings for a Page URL, but we are unable to discover how to actualy access that page via that URL. Is this documented anywhere?

**Comment by nimitshukla123** [comment]
you can access the page URL like this

basePath/${Page URL in page setting}/${pageID.html}

On Wed, Jul 17, 2019 at 9:19 AM Ezra Sharp <notifications@github.com> wrote:

> There is a field in the page settings for a Page URL, but we are unable to
> discover how to actualy access that page via that URL. Is this documented
> anywhere?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/SalesforceCommerceCloud/teams/community/discussions/86?email_source=notifications&email_token=ADMLMX44VZJW3R452EP2GMDP7ZM3BA5CNFSM4IEI65L2YY3PNVWWK3TUL52HS4DFVZCGS43DOVZXG2LPNZIG643UVJRW63LNMVXHIX3JMTHAAAI5XE>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADMLMX2MNESNYQVEPU6SM3LP7ZM3BANCNFSM4IEI65LQ>
> .
>


-- 

Regards



*Nimit Shukla*

T: +65 84575208
